test/docs/__snapshots__/get-docs-yaml.api-test.ts.snap

```diff
@@ -205,6 +238,67 @@ components:
         - citizenshipClass
         - officerRiskDate
         - countryCode
+    CreatePartyRequestItem:
+      type: object
+      properties:
+        alternateIdentifier:
+          type: string
+          description: The UKEF ID for the party. Should contain just digits.
```

Comment:
I think we could technically add a `format` option here that specifies some regex it should match. Then we could enter the regex for "exactly 8 digits". That might be overkill though - what do you think?

Reply:
I think it's `pattern` rather than `format` (https://swagger.io/docs/specification/data-models/data-types/) - I've added `pattern: /^d{8}$/`